### ISSUE TITLE
Command and Control

### ISSUE THREAD

**Issue by Jethro-San** [BruceDevices/firmware#498]
This sounds a little ridiculous even to me, but would it be possible to implement a Mirai type botnet command and control feature, or some sort of command-and-control server with a bad USB script to enter the device into the server via the command prompt and include some features such as remote Rickrolling and access to command prompt on the system.

**Comment by 9dl** [comment]
Do you mean a C2 server where you log in via SSH, such as using PuTTY?

**Comment by Jethro-San** [comment]
not necessarily, something like https://github.com/offsecginger/koadic where somehow something has to be entered into the command prompt on the system to connect to the server, sometimes it works with SSH, but often it works by running some bash on the target machine

**Comment by 9dl** [comment]
Oh okey, I can look into it.
